pkg/lwip/contrib/sock/ip/lwip_sock_ip.c

```diff
@@ -189,62 +199,146 @@ ssize_t sock_ip_recv_aux(sock_ip_t *sock, void *data, size_t max_len,
     ssize_t res, ret = 0;
     bool nobufs = false;
 
+    bool peek = false;
+    if (aux != NULL) {
+        if (IS_USED(MODULE_SOCK_AUX_PEEK)) {
+            peek = aux->flags & SOCK_AUX_PEEK;
+        }
+    }
+
     assert((sock != NULL) && (data != NULL) && (max_len > 0));
-    while ((res = sock_ip_recv_buf_aux(sock, &pkt, (void **)&ctx, timeout,
-                                       remote, aux)) > 0) {
+    while ((res = sock_ip_recv_buf_aux(sock, &pkt, (void**)&ctx, timeout, remote,
+                                       aux)) > 0) {
```

Comment:
No need to change that :wink: 
```suggestion
    while ((res = sock_ip_recv_buf_aux(sock, &pkt, (void **)&ctx, timeout,
                                       remote, aux)) > 0) {
```

Reply:
Some artifact from previous edits, I'll undo it